### DISCUSSION TITLE
加密后无法正常运行

### DISCUSSION THREAD

**Discussion by hayingbs** [dashingsoft/pyarmor#1736] [Q&A]
求大神帮看看啥问题
![未标题-3](https://github.com/dashingsoft/pyarmor/assets/99177908/762e00f4-b301-4ffc-bacd-d75987c590fa)
![未标题-1](https://github.com/dashingsoft/pyarmor/assets/99177908/e0d97dc6-69a9-4a3f-81ea-f340fe78d62b)

**Comment by kaysvip** [comment]
我也遇到 直接崩溃了

**Reply by wjl130825** [reply]
PyInstaller\loader\pyimod02_importers.py", line 384, in exec_module
  File "pyarmor_runtime_005595\__init__.py", line 2, in <module>
RuntimeError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。 (1:11076)

**Reply by wjl130825** [reply]
我的提示这个问题

**Comment by hayingbs** [comment]
先打包后加密
